google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/features/InstanceApi.java

```diff
@@ -250,7 +251,8 @@ Operation setDiskAutoDelete(@PathParam("instance") String instanceName,
    @MapBinder(BindToJsonPayload.class)
    Operation setScheduling(@PathParam("instance") String instanceName,
                            @PayloadParam("onHostMaintenance") Scheduling.OnHostMaintenance onHostMaintenance,
-                           @PayloadParam("automaticRestart") boolean automaticRestart);
+                           @PayloadParam("automaticRestart") boolean automaticRestart,
+                           @PayloadParam("preemptible") boolean preemptible);
```

Comment:
Just overload the method and still provide one without the preemptible parameter? If needed, for the *old* method we could have a specialised binder extending the BindToJsonPayload that would set the preemptible flag to false, but I doubt that would be needed. It looks like "false" was already the default if the parameter was not present in the body.